golf2/src/main.rs

```diff
@@ -386,23 +386,23 @@ pub unsafe fn reset_handler() {
     }
 
     let mut _ctr = 0;
-    let chip = static_init!(h1b::chip::Hotel, h1b::chip::Hotel::new());
+    let chip = static_init!(h1::chip::Hotel, h1::chip::Hotel::new());
     chip.mpu().enable_mpu();
 
     let end = timerhs.now();
     println!("Tock: booted in {} tics; initializing USB and loading processes.",
              end.wrapping_sub(start));
 
-    h1b::usb::USB0.init(&mut h1b::usb::EP0_OUT_DESCRIPTORS,
-                        &mut h1b::usb::EP0_OUT_BUFFERS,
-                        &mut h1b::usb::EP0_IN_DESCRIPTORS,
-                        &mut h1b::usb::EP0_IN_BUFFER,
-                        &mut h1b::usb::EP1_OUT_DESCRIPTOR,
-                        &mut h1b::usb::EP1_OUT_BUFFER,
-                        &mut h1b::usb::EP1_IN_DESCRIPTOR,
-                        &mut h1b::usb::EP1_IN_BUFFER,
-                        &mut h1b::usb::CONFIGURATION_BUFFER,
-                        h1b::usb::PHY::A,
+    h1::usb::USB0.init(&mut h1::usb::EP0_OUT_DESCRIPTORS,
+                        &mut h1::usb::EP0_OUT_BUFFERS,
+                        &mut h1::usb::EP0_IN_DESCRIPTORS,
+                        &mut h1::usb::EP0_IN_BUFFER,
+                        &mut h1::usb::EP1_OUT_DESCRIPTOR,
+                        &mut h1::usb::EP1_OUT_BUFFER,
+                        &mut h1::usb::EP1_IN_DESCRIPTOR,
+                        &mut h1::usb::EP1_IN_BUFFER,
+                        &mut h1::usb::CONFIGURATION_BUFFER,
+                        h1::usb::PHY::A,
```

Comment:
The alignment of lines 396-405 is off by 1.

Reply:
Oops, well spotted.